python-avd/pyavd/_cv/workflows/finalize_workspace_on_cv.py

```diff
@@ -40,8 +40,12 @@ async def finalize_workspace_on_cv(workspace: CVWorkspace, cv_client: CVClient)
 
     workspace_config = await cv_client.build_workspace(workspace_id=workspace.id)
     build_result, cv_workspace = await cv_client.wait_for_workspace_response(workspace_id=workspace.id, request_id=workspace_config.request_params.request_id)
+    workspace.build_id = cv_workspace.last_build_id
     if build_result.status != ResponseStatus.SUCCESS:
         workspace.state = "build failed"
+        ws_build_errors = await process_workspace_build_errors(workspace_id=workspace.id, build_id=workspace.build_id, cv_client=cv_client)
```

Comment:
Even for a successful build we could get warnings. I think we should get these build results no matter what and then log out warnings and errors (which will be picked up correctly by ansible as well)

Reply:
We could consider adding a knob to avoid logging the warnings.

Reply:
Changed behavior to always pull build details with ability to suppress:
* all warnings
* specific warnings based on a known behavior (like repeatable warnings regarding enabling `portfast` on a `switchport`)
* warnings matching custom string patterns controlled by the user